lib/ipmi_sensor.c

```diff
@@ -169,7 +169,25 @@ ipmi_sensor_print_fc_discrete(struct ipmi_intf *intf,
 	}
 
 	if (csv_output) {
-		/* NOT IMPLEMENTED */
+        printf("%s", sr->s_id);
+        if (sr->s_reading_valid) {
+            if (sr->s_has_analog_value) {
+                /* don't show discrete component */
+                printf(",%s%s%s",
```

Comment:
Are you sure that, for instance, "0.97Vok"  is what you want to have in a cell? I believe there must be either spaces or field separators between those `%s` specifiers.

Anyway, this won't work unless you enclose values in double quotes as per [RFC4180](https://tools.ietf.org/html/rfc4180) because some locales use comma as a decimal point.